tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "removeComments": false,
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "noEmitOnError": true,
+    "allowJs": false,
+    "jsx": "react",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "target": "es6",
+    "module": "commonjs",
+    "noEmit": false,
+    "declaration": false,
+    "strictNullChecks": false,
+    "outDir" : "./.tsbuild"
+  },
+  "files": [
+    "typings/custom.d.ts"
```

Comment:
I'd also set `typingsRoot` to help the compiler find typings.

Reply:
according to the docs, if you set typingsRoot it blows away the default settings whereby it picks up everything under node_modules, so I could add it but would have to put back the defaults into my config. Prob easier to just keep a single pointer to my custom typings. Unless I'm missing someting.